package.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name": "bitfinex-promise",
-	"version": "1.0.5",
+	"version": "1.0.6",
 	"description": "node.js wrapper for bitfinex cryptocurrency exchange using promises",
 	"main": "lib/Bitfinex.js",
     "license": "MIT",
 	"scripts": {
 		"build": "coffee --compile --output lib/ src/*.coffee",
 		"prepublish": "coffee --compile --output lib/ src/*.coffee",
 		"test": "mocha",
+		"test-compile": "coffee --compile --output test/ src-test/*.coffee",
```

Comment:
Cool, now just add that to the test line. =) Sorry I wasn't clear enough. I'm fine with forcing a compile every time tests are run. =)

`"test": "npm run build && npm run test-compile && mocha"`